src/counting/classical_tree_counter.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+
+class TreeCounter:
+
+    def __init__(self, *args, **kwargs):
+        '''
+        Any required arguments for the algorithm 
+        that stay unchanged for every run 
+        on every forest part, and any required
+        initialisation.
+        '''
+        pass
+
+    def count(self, rgb_image: np.ndarray, ndvi_image: np.ndarray,
+              forest_mask: np.ndarray):
+        assert 3 == len(rgb_image.shape), \
+            "RGB image array should be 3-dimensional"
+        assert 2 == len(ndvi_image.shape), \
+            "NDVI image array should be 2-dimensional"
```

Comment:
@mkosturek  NDVI chyba nie jest potrzebny do zliczania drzew jak to robimy pattern matchingiem

Reply:
OK, a czy myślisz, że nie będzie to nigdy potrzebne?